docs/site/Validation-ORM-layer.md

```diff
@@ -0,0 +1,14 @@
+---
+lang: en
+title: 'Validation in ORM Layer'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Validation-ORM-layer.html
+---
+
+The validation in the ORM layer is to make sure the data being added or updated
+to the database is valid.
+
+There is validation coming from the legacy juggler. The validation is enforced
```

Comment:
Perhaps provide a link to legacy juggler?

Reply:
> There is validation coming from the legacy juggler

There is  some validation done in  legacy juggler.

Reply:
> The validation is enforced at the database level, there is not much additional constraints we can set
beyond the database ones 

The validation is enforced at the database level, so there aren't many additional constraints we can set
beyond the database ones.

Reply:
We can mention schema constraints enforced by specific DBs, such as unique index.

For preprocessing against CRUD operations, we can refer to https://loopback.io/doc/en/lb4/migration-models-operation-hooks.html.